pages/index.js

```diff
@@ -5,6 +5,7 @@ import { FiGithub } from "react-icons/fi";
 import Card from "../components/card";
 import Footer from "../components/footer";
 import HeadObject from "../components/head";
+import RepoData from "../data/PROJECTS.json";
```

Comment:
why in gods green world is this capitalized

Reply:
So that I can remember to remove it later